app/assets/stylesheets/govuk_publishing_components/components/govspeak/_steps.scss

```diff
@@ -11,13 +11,32 @@
       list-style-type: decimal;
       margin-left: 0;
       padding: .75em 0 .75em 2.5em;
+      position: relative;
+
+      // IE doesn't render this correctly with this elemeent as display:block
+      // (it leaves a space for the :before pseudo element) by setting to
+      // display inline block and a width of 100% it works correctly.
+      display: inline-block;
+      width: 100%;
```

Comment:
This seems like a good solution. My only concern is that with padding to the left of the element, a width of 100% means it extends beyond its container, as shown below. I can't think of a situation where this might cause an actual problem, but it might be worth applying `box-sizing: border-box;` to be sure.

![Screenshot 2023-01-03 at 09 42 36](https://user-images.githubusercontent.com/861310/210332981-5f228a3a-594a-4174-9051-b3b37f0fdf66.png)

Reply:
Ah yes, good shout on the border box. I've added that in.

Do you happen to have any experience with the problem I faced? It seems that in IE when I'm using the `:before` pseudo-element it continues to impact the layout of the li despite having `position: absolute`. I failed to find anything online of similar issues so don't know if it's something common or not. It was quite baffling.